### ISSUE TITLE
display database usage like ios version.

### ISSUE THREAD

**Issue by liorbraginsky92** [nextcloud/android#2874]
### Actual behaviour
- Tell us what happens

### Expected behaviour
- Tell us what should happen
 
### Steps to reproduce
1. 
2. 
3. 

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by tobiasKaminsky** [comment]
@liorbraginsky92 what is missing from your side?

We have in sidebar, without quota:
![2018-10-23-102855](https://user-images.githubusercontent.com/5836855/47346919-74617e00-d6ae-11e8-9a14-01c914abbf86.png)

With quota:
![2018-10-23-102944](https://user-images.githubusercontent.com/5836855/47346981-95c26a00-d6ae-11e8-925f-b76a868da667.png)

Same will be soon shown in Account info.

**Comment by AndyScherzinger** [comment]
shipped with 3.3.x or will ship with 3.4. thus closing
